packages/ra-ui-materialui/src/input/AutocompleteInput.tsx

```diff
@@ -446,9 +446,9 @@ const AutocompleteInput: FunctionComponent<AutocompleteInputProps> = props => {
                     color,
                     ...inputProps
                 } = getInputProps({
+                    ...InputProps,
```

Comment:
Why did you make this (breaking) change? It prevents developers from overriding `onBlur` and `onFocus`